modules/isc_dhcpd/isc_dhcpd.go

```diff
@@ -0,0 +1,97 @@
+package isc_dhcpd
+
+import (
+	"github.com/netdata/go.d.plugin/pkg/ip"
+
+	"github.com/netdata/go.d.plugin/agent/module"
+)
+
+type (
+	Config struct {
+		LeaseFile string            `yaml:"leases_path"`
+		Pools     map[string]string `yaml:"pools"`
+		Dim       map[string]Dimensions
+	}
+
+	Dimensions struct {
+		Values ip.IRange
+		Name  string
+	}
+)
+
+type DHCPD struct {
+	module.Base
+	Config `yaml:",inline"`
+
+	collectedLeases bool
+	charts          *module.Charts
+}
+
+func init() {
+	module.Register("isc_dhcpd", module.Creator{
+		Defaults: module.Defaults{
+			UpdateEvery: 10,
```

Comment:
we should use update_every 1, but to not parse file every second we need to the file check modification time, if it is changed - we parse it.

Reply:
I already did a partial change in this direction and I am checking the file before to parse it. After to rewrite the parser, everything will be completed.